packages/core/src/browser/widget-manager.ts

```diff
@@ -114,7 +114,7 @@ export class WidgetManager {
 
     protected _cachedFactories: Map<string, WidgetFactory>;
     protected readonly widgets = new Map<string, Widget>();
-    protected readonly widgetPromises = new Map<string, MaybePromise<Widget>>();
+    // protected readonly widgetPromises = new Map<string, MaybePromise<Widget>>();
```

Comment:
Minor: I don't believe this serves any real purpose, right?

Reply:
It didn't before - apart from causing errors - and it certainly doesn't now :-).